pkg/ucp/frontend/controller/planes/listplanes.go

```diff
@@ -54,28 +54,43 @@ func (e *ListPlanes) Run(ctx context.Context, w http.ResponseWriter, req *http.R
 	serviceCtx := v1.ARMRequestContextFromContext(ctx)
 	logger := ucplog.FromContextOrDiscard(ctx)
 
-	query := store.Query{
-		RootScope:    serviceCtx.ResourceID.String(),
-		IsScopeQuery: true,
+	// The plane objects are all stored separately (by plane type). We need to query each type separately.
+	planeTypes := []string{
```

Comment:
Should we put this in the `types.go` or something like that and call it **availablePlaneTypes**?

Reply:
If we end up with a second location where we need this list then sure.